### ISSUE TITLE
Add corrections for multiplicity to multiway ANOVA

### ISSUE THREAD

**Issue by EJWagenmakers** [jasp-stats/jasp-desktop#1014]
@raviselker: it would be nice to add the possibility to correct p-values from multiway ANOVA for multiplicity. See http://arxiv.org/abs/1412.3416 and https://github.com/singmann/afex/pull/3. Another way to add something that SPSS lacks. I think this option should not be default, but available through a checkbox instead. 
Cheers, E.J.

**Comment by R180** [comment]
Yes this is the well-known familywise error problem. And yet, we could just as well try to correct for familywise error across all studies we've ever conducted. Why not simply accept that we've chosen a particular tolerance for Type 1 errors? With a fixed alpha, the number of errors (but not the proportion of errors) will naturally rise as more tests are conducted. But that's exactly what we should want, if we want alpha to be fixed.

**Comment by IndrajeetPatil** [comment]
Just read Dan Lakes post on this issue and was wondering the same:
http://daniellakens.blogspot.nl/2016/01/error-control-in-exploratory-anovas-how.html

This would definitely be a welcome addition to JASP.

**Comment by EJWagenmakers** [comment]
@raviselker @IndrajeetPatil Yes, I agree this would be a nice addition.  @R180 : I agree the general problem is well-known, but the fact that it is also present in multiway ANOVAs (instead of one-way ANOVAs) is definitely underappreciated.
